vnext/Microsoft.ReactNative/ReactNative.V8JSI.Windows.WinUI3.targets

```diff
@@ -0,0 +1,21 @@
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemDefinitionGroup>
+    <Link>
+      <AdditionalLibraryDirectories Condition="'$(Configuration)' == 'WinUI3' And '$(Platform)' == 'x64'">$(V8_Package)\build\native\..\..\lib\Debug\x64;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories Condition="'$(Configuration)' == 'WinUI3' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">$(V8_Package)\build\native\..\..\lib\Debug\x86;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalDependencies>v8jsi.dll.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
>v8jsi.dll.lib [](start = 30, length = 13)

The Microsoft.ReactNative.dll must not depend or use the v8jsi.dll. This DLL does not pass the UWP store validation and any application that use it will not be deployed to the Microsoft store.
This DLL is only for Win32 apps.

Reply:
interesting... I added this because the project tries to link against v8jsi and it was complaining about it. I wonder if we then are wrongly linking v8jsi always? will investigate more

Reply:
@vmoroz Ok confirmed that this is at parity with Debug/Release today: we don't end up loading v8jsi.dll at runtime, the only thing that gets linked in is v8jsi.dll.**lib**. This is true for regular debug/release builds too. USE_V8 is true for everything except ARM/ARM64:

C:\rnw2\vnext\PropertySheets\React.Cpp.props

    <USE_V8 Condition="'$(USE_V8)' == '' AND '$(Platform)' != 'ARM' AND '$(Platform)' != 'ARM64'">true</USE_V8>

But the dll isn't actually loaded at runtime. If this needs to be changed in any way, please file an issue and let's do it in a separate PR.